misskey-util/src/client.rs

```diff
@@ -3047,6 +3046,209 @@ pub trait ClientExt: Client + Sync {
     }
     // }}}
 
+    // {{{ Page
+    /// Creates an empty page with the given name.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use misskey_util::ClientExt;
+    /// # #[tokio::main]
+    /// # async fn main() -> anyhow::Result<()> {
+    /// # let client = misskey_test::test_client().await?;
+    /// let page = client.create_page("name").await?;
+    /// assert_eq!(page.name, "name");
+    /// # client.delete_page(&page).await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    fn create_page(&self, name: impl Into<String>) -> BoxFuture<Result<Page, Error<Self::Error>>> {
```

Comment:
imo, this does not have much use case and having only `build_page` is sufficient